blockchain/treasury_policy_test.go

```diff
@@ -0,0 +1,538 @@
+package blockchain
```

Comment:
FYI there is a typo in the commit message for moving this file: "This is purely a code **move**."